client/src/app/site/pages/organization/pages/accounts/pages/account-list/components/account-list/account-list.component.html

```diff
@@ -100,6 +100,10 @@ <h2>{{ 'Accounts' | translate }}</h2>
             <mat-icon>edit</mat-icon>
             <span>{{ 'Edit' | translate }}</span>
         </a>
+        <a mat-menu-item [routerLink]="[member.id, 'meetings']">
+            <mat-icon>auto_awesome_mosaic</mat-icon>
+            <span>{{ 'Add to meetings' | translate }}</span>
+        </a>
```

Comment:
same here, some check should exist so it is not displayed to users without permission (or is this whole view only accessible with the correct OML? then nvm)